cmd/porter/inspect.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/deislabs/porter/pkg/porter"
+)
+
+func buildBundleInspectCommand(p *porter.Porter) *cobra.Command {
+	opts := porter.ExplainOpts{}
+	cmd := cobra.Command{
+		Use:   "inspect",
+		Short: "Inspect a bundle",
+		Long:  "Inspect a bundle by printing the parameters, credentials, outputs, actions and images.",
```

Comment:
If the bundle has been copied elsewhere and has a relocation map, will this include that information too?

Reply:
It currently shows the relocated images. It could show both I suppose? I'm not really a fan of displaying just the relocation map as is, it just shows two images, doesn't easily tie back to invocation images vs images, so I just printed out the relocated version, if it exists. I guess we could add another column and show original and new.